### ISSUE TITLE
Not Working With GBS Control

### ISSUE THREAD

**Issue by c64manguy** [jtsiomb/gbs-cgaega#1]
Hey All. 
Has anyone gotten this working with a GBS-Control modded gbs8200 or 8220? I had 10 boards built and soldered on the components. I programmed the gal with cgaega.jed and when I turn the computer on (IBM XT 286) with an EGA card I just get a bunch of white scrambled up nonsense jumping around the top of the screen...

**Comment by jtsiomb** [comment]
I haven't tried gbs-control yet. I would assume it's orthogonal, that gbs-control would just enable accepting 21khz sync signals. But maybe it also changes the sync required to separate H/V sync instead of combined H+V ? Or is there an option for this in gbs-control? If you figure it out let us know.

**Comment by c64manguy** [comment]
Have you had it working on a normal GBS8200 or 8220? With out the GBS-Control mods?

**Comment by jtsiomb** [comment]
Yes, mine is a GBS 8200. I don't know the differences between that and the 8220.

**Comment by c64manguy** [comment]
Im going to try it with a regular GBS-8200 and see what happens. My EGA card could be bad. With the GBS-Control I have been able to get a dim output as long as the EGA card isnt set to High Res Mode. Can you tell me what the CS and NOT-CS are for? And what they should be set to?

**Comment by c64manguy** [comment]
I hooked it up with a standard GBS8200 board and it worked with text but when I tried to load Windows 3.1 it lost sync. Not sure if my EGA card is messed up or not. Can you load Windows 3.1 on yours or color modes in general or are you using yours for something else?

**Comment by jtsiomb** [comment]
Not easily. I've got the CGA card on an 8088 XT clone without a hard disk. I could try some CGA game which runs from floppy to test, but it will take a while, as I've got all that hardware stashed at the moment.

**Comment by c64manguy** [comment]
Oh, I wasn't asking you to try it yourself. Just wondering what you used yours for. You've done plenty just making the board for everyone to use. Thank you. 
I ordered the parts to make a sync stripper. I'll see if that helps.

**Comment by banman11** [comment]
Hi very interesting discussion. I think the difference between the GBS 8200 and GBS 8220 boards is the inclusion of a second VGA output adapter.

**Comment by theos10** [comment]
Hi my cga computer not work with gbs-8200 moded or not, and samsung 151mp display

**Comment by jtsiomb** [comment]
Since this original discussion I have modded my GBS-8200 with gbs-control, and it works perfectly fine with my CGA/EGA input board.

EGA 21khz modes however are very hard to get to sync right, but CGA modes work and are much crisper than with the unmodified GBS board.

Here's a screenshot of my termtris game running on EGA 21khz. It mostly syncs after a fair bit of tweaking, but it looses it once in a while, and always distorts the top of the screen:
![termtris_ega](https://github.com/user-attachments/assets/59739c31-51cf-48d0-8e35-1412984d419e)

edit: this is pretty much best case for 21khz, and mostly seems to work because termtris is black at the bottom. Trying it with something like norton commander, the bottom of the screen also folds up and overlaps with the rest in a strange way if I remember correctly. It's not very usable. At least about a year ago when I tried this, not sure if the gbs-control people managed to fix this somehow since then.

**Comment by banman11** [comment]
Hi jtsiomb,

Looks interesting, good job.


I haven't had time to revisit this discussion.

However, some time back I purchased this cheap GBS 8200 (I assume it is)....


AU$33.79 | Arcade Game RGB/CGA/EGA/YUV To VGA HD Video Converter Board HD9800/GBS8200 Hot Green Board
https://a.aliexpress.com/_ms7itXw


Now I had a great deal of issues with screen trearing/ sync? with my c128 setup.
Quite by accident I discovered that by momentarily shorting out a jumper on the GBS board the issue disappeared entirely.

I am no expert on this. I think shorting out the jumper disables the on-screen menu processor which is causing the tearing/ sync issues.

NOTE: shorting this jumper out disables the on-screen menu. Starting the GBS board resets it back to it's factory state. 

On my GBS board it's the only 2 pin jumper there, easy enough to tell..

Maybe this is of interest to people having screen issues.

**Comment by banman11** [comment]
![Screenshot_2024-09-06-06-45-45-175_cn wps moffice_eng](https://github.com/user-attachments/assets/c5cfcd69-513e-44d7-a347-5f022009c38d)

![Screenshot_2024-09-06-06-55-38-667_cn wps moffice_eng](https://github.com/user-attachments/assets/3ede162f-a7e5-402b-8d5a-58d203a9f953)

**Comment by banman11** [comment]
Looks like this different GBS variant has one as well.


![Screenshot_2024-09-06-06-51-12-001_cn wps moffice_eng](https://github.com/user-attachments/assets/2f5d3561-07f8-43fb-925f-ae0f057f323d)

**Comment by Verault** [comment]
Does anyone have one of these boards to sell or know if someone is selling them. I would really like to build one but dont need to order a bunch of boards just to make one.

**Comment by jtsiomb** [comment]
> Does anyone have one of these boards to sell or know if someone is selling them. I would really like to build one but dont need to order a bunch of boards just to make one.

The minimum order at JLCPCB is 5 boards, and it costs like $2. Build one and throw the other 4 away.

**Comment by Verault** [comment]
My OCD would never let me do that.
